website/content/docs/secrets/azure.mdx

```diff
@@ -37,17 +37,49 @@ management tool.
 
 1. Enable the Azure secrets engine:
 
-   ```shell
+<Tabs>
+
+<Tab heading="Vault CLI">
+
+   ```shell-session
    $ vault secrets enable azure
    Success! Enabled the azure secrets engine at: azure/
    ```
 
    By default, the secrets engine will mount at the name of the engine. To
    enable the secrets engine at a different path, use the `-path` argument.
 
+</Tab>
+
+<Tab heading="Vault UI">
+
```

Comment:
We should use screenshots sparingly. Generally, we want to pick one "hero" screenshot that helps folks orient themselves in the GUI to make sure they're looking at the right screen.